product-service/lib/lambdas/createProduct.ts

```diff
@@ -0,0 +1,48 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { formatResponse } from './utils';
+import  * as ProductService from './services/ProductService';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  try {
+    console.log('Processing createProduct request');
```

Comment:
In requirement `All lambdas do console.log for each incoming requests and their arguments`. It might be necessary to add certain arguments to the log, for example, request payload.